src/domain/permissions.js

```diff
@@ -36,7 +37,8 @@ const rolesForPermission = {
   [MANAGE_IMPORT]: [ROLE.admin],
   [OWN_ROOMS]: [ROLE.admin, ROLE.user],
   [AUTORIZE_ROOMS_RESOURCES]: [ROLE.admin, ROLE.user],
-  [JOIN_PRIVATE_ROOMS]: [ROLE.admin, ROLE.user]
+  [JOIN_PRIVATE_ROOMS]: [ROLE.admin, ROLE.user],
+  [REGENERATE_ALL_DOCS]: [ROLE.admin]
```

Comment:
Small thing, I'd shorten this and drop the "all" here, to me it brings no extra info cause it's implied, just like with the other permissions (e.g "VIEW_DOCS", "EDIT_SETTINGS" etc.)

Reply:
I thought it emphasize that it's a rather special thing (we might have just "regenerate_docs" for lower role users, but true, we can introduce that if needed later). I'll change